html-templates/subtemplates/forms.tpl

```diff
@@ -69,3 +69,4 @@
 
 {template loginField}{field inputName=_LOGIN[username] label=Username required=true attribs='autofocus autocapitalize="none" autocorrect="off"' hint='You can also log in with your email address.'}{/template}
 {template passwordField}{field inputName=_LOGIN[password] label=Password hint='<a href="/register/recover">Forgot?</a>' required=true refill=false type=password}{/template}
+{template rememberMeField}{if $.cookies['username'] !== null}{$atts = 'CHECKED'}{/if}{checkbox _LOGIN[remember] '' attribs=$atts label='Remember Me'}{/template}
```

Comment:
@jtwiest use the more concise dwoo array syntax `$.cookies.username`

Also for the use of `{checkbox}` don't use unnamed parameters